recipe/bld.bat

```diff
@@ -1,5 +1,5 @@
 @echo ON
 setlocal enabledelayedexpansion
 
-make install PREFIX=%LIBRARY_PREFIX%
+make install PREFIX=%LIBRARY_PREFIX% LCOV_PERL_PATH=
```

Comment:
1.14 upstream uses `"#!/usr/bin/env perl"` which is what our patch used to do, but then it introduced a install time "feature" to overwrite the shebang with `LCOV_PERL_PATH` defaulted to `/usr/bin/perl`.  Suppressing this behavior.